content/source/docs/cloud/run/ui.html.md

```diff
@@ -86,6 +86,8 @@ Whenever a pull request is _created or updated,_ Terraform Cloud checks whether
 
     -> **Note:** If Terraform Cloud skips a plan because the changes weren't relevant, it will still post a passing commit status to the pull request.
 
+- Status checks on the pull request are not updated after an apply, so a commit with a successful plan but an errored apply will still show the passing commit status relevant to the plan.
```

Comment:
Hmm, took me a few passes to understand that, but it does seem like an important thing to point out. How about this? 

```suggestion
- Commit statuses only show the results of speculative plans. If a pull request is merged and results in a normal (non-speculative) run, Terraform Cloud will not update the pull request's status checks with the results (for example, if an apply partially fails).
```

Or that might prompt you to think of a better way to describe that; either way, I'll go ahead and pre-approve.